articles/azure-large-instances/what-is-azure-for-large-instances.md

```diff
@@ -0,0 +1,167 @@
+---
+title: What is Azure for Large Instances?
+description: Provides an overview of Azure for Large Instances.
+author: jjaygbay
+ms.author: jacobjaygbay
+ms.service: baremetal-infrastructure
+ms.custom: references_regions
+ms.topic: conceptual
+ms.date: 06/01/2023
+---
+
+#  What is Azure for Large Instances?
+
+Microsoft Azure offers a cloud infrastructure with a wide range of integrated cloud services to meet your business needs. 
+In some cases, though, you may need to run services on Azure Large Instances without a virtualization layer. 
+You may need root access and control over the operating system (OS). To meet this need, Azure offers Azure Large Instances for several high-value, mission-critical applications. 
+
+Azure Large Instances (ALI) is made up of dedicated large instances (compute instances). It features: 
+
+* High-performance storage appropriate to the application (Fiber Channel). Storage can also be shared across Azure Large Instances to enable features like scale-out clusters or high availability pairs with failed-node-fencing capability. 
+
+* A set of function-specific virtual LANs (VLANs) in an isolated environment. 
+
+This environment also has special VLANs you can access if you're running virtual machines (VMs) on one or more Azure Virtual Networks (VNets) in your Azure subscription. 
+The entire environment is represented as a resource group in your Azure subscription. 
+
+Azure Large Instances is offered in over 30 SKUs from 2-socket to 24-socket servers and memory ranging from 1.5 TBs up to 24 TBs. 
```

Comment:
@vasthomsft and @gosundar: There has been a comment from engineering lead Hrushi on the need of exposing the details of various SKUs supported by ALI in general and the memory range.Should we take this off for Day 1 of documentation publication ?